src/convertToObject.js

```diff
@@ -11,8 +11,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
   // write your code here
+  const convertedObject = sourceString
+    .split(';')
+    .filter(property => property.length > 4)
+    .reduce(function(CSSObject, property) {
+      const propPair = property.split(':');
+
+      CSSObject[propPair[0].trim()] = propPair[1].trim();
+
+      return CSSObject;
+    }, {});
```

Comment:
Use arrow functions as callback, it looks shorter and more readable 
```suggestion
    .reduce((CSSObject, property) => {
      const propPair = property.split(':');

      CSSObject[propPair[0].trim()] = propPair[1].trim();

      return CSSObject;
    }, {});
```